docker-compose/server.config.yml

```diff
@@ -3,16 +3,22 @@ springdoc:
     path: /v1/swagger
 ldes-server:
   host-name: "http://localhost:8080"
+  compaction-cron: 0 * * * * *
```

Comment:
I wouldn't add this to the default server config. At least not with a frequency this high.